src/oss/confluent.sh

```diff
@@ -686,6 +755,43 @@ config_service() {
     fi
 }
 
+enable_metrics_reporter() {
+    is_enterprise
+    status=$?
+    if [[ ${status} -ne 0 ]]; then
+        return 1
+    fi
+
+    local service="${1}"
+
+    local service_dir="${confluent_current}/${service}"
+    echo "metric.reporters=io.confluent.metrics.reporter.ConfluentMetricsReporter" \
```

Comment:
I think C3 has two properties file, one of which is `-dev`.  Is there value in having the same thing for `server.properties`?  ie, `server-dev.properties`?

Reply:
I believe such generalization in packaging gets out of the scope of this PR. The other services work sufficiently with their current configs. My suggestion is that we should reconsider such a change in a later release.